chia/util/struct_stream.py

```diff
@@ -89,4 +89,4 @@ def stream_to_bytes(self) -> bytes:
 
     # this is meant to avoid mixing up construcing a bytes object of a specific
     # size (i.e. bytes(int)) vs. serializing the integer to bytes (i.e. bytes(uint32))
-    __bytes__ = None
+    __bytes__: None = None
```

Comment:
This was changed in release and then checkpointed so presumably it is just conflicted.  It's a regular function def again (for now) so probably just revert this.

```suggestion
    __bytes__ = None
```